auto-generated/cluster/stone-prd-rh01/tenants/rhtap-contract-tenant/appstudio.redhat.com_v1alpha1_releaseplan_ec-v01-alpha-registry-redhat-io.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: "true"
+    release.appstudio.openshift.io/standing-attribution: "true"
```

Comment:
Do we need `release.appstudio.openshift.io/author` here?

Reply:
I don't see it on other release plans, so I guess no..?

Reply:
See also [these docs](https://docs.google.com/document/d/1nCHMYxyJQW_fAH8z97uk16IvYTQaodbUn87fMGgw8YA/edit#heading=h.4p8list4bfyj).